pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -107,7 +104,12 @@ func BuildKubernetesMasterConfig(options configapi.MasterConfig, requestContextM
 		return nil, err
 	}
 
-	admissionController := admission.NewFromPlugins(kubeClient, strings.Split(server.AdmissionControl, ","), server.AdmissionControlConfigFile)
```

Comment:
they no longer have the option to override via args... I assume that's why we were setting the value into the server before